src/mlx90632.c

```diff
@@ -449,4 +471,145 @@ int32_t mlx90632_init(void)
 
     return 0;
 }
+
+int32_t mlx90632_addressed_reset(void)
+{
+    int32_t ret;
+
+    ret = mlx90632_i2c_write(0x3005, MLX90632_RESET_CMD);
+    if (ret < 0)
+        return ret;
+
+    usleep(150, 200);
+
+    return 0;
+}
+
+
+/** Reads the refresh rate and calculates the time needed for a single measurment from the EEPROM settings.
+ *
+ * @param[in] meas Measurement to read the frefresh rate for
+ *
+ * @retval >= Refresh time in ms
+ * @retval <0 Something went wrong. Check errno.h for more details.
+ */
+static int mlx90632_get_measurement_time(uint16_t meas)
+{
+    int32_t ret;
+    uint16_t reg;
+
+    ret = mlx90632_i2c_read(meas, &reg);
+    if (ret < 0)
+        return ret;
+
+    reg &= MLX90632_EE_REFRESH_RATE_MASK;
+    reg = reg >> 8;
+
+    return MLX90632_MEAS_MAX_TIME >> reg;
+}
+
+/** Reads the refresh rate and calculates the time needed for a whole measurment table from the EEPROM settings.
+ *
+ * The function is returning valid measurement time only for burst mode measurements.
+ * An error will be returned if it is called with a continuous measurement type parameter.
+ *
+ * @param[in] burst_meas_type Measurement type to read the frefresh rate for
+ *
+ * @retval >= Refresh time in ms
```

Comment:
Strange one... if its retval its usually some value, not two signs. I assume

```suggestion
 * @retval >=0 Refresh time in ms
```